force-app/main/default/classes/OpportunityServiceTest.cls

```diff
@@ -0,0 +1,28 @@
+@isTest
+private class OpportunityServiceTest {
+    @TestSetup
+    private static void createTestData(){
+        Account[] accList = TestDataFactory.createAccounts(5, true);
+        Opportunity[] oppList = TestDataFactory.createOpportunities(accList, 6, true);
+        Integer accNum = [SELECT COUNT() FROM Account];
+        Integer oppNum = [SELECT COUNT() FROM Opportunity];
+        System.assertEquals(5, accNum,'Count of records should be equal account input.');
+        System.assertEquals(30, oppNum,'Count of records should be equal to opps per acc x # of accounts.');
+    }
+
+    @isTest private static void getOpportunitiesById(){
+        List<Account> oppsFromAcc = [
+            SELECT Name, Id, (SELECT Name, StageName, CloseDate FROM Opportunities ORDER BY StageName, CloseDate DESC)
+            FROM Account
+            LIMIT 1
+        ];
+        System.assert(oppsFromAcc.size() == 1, 'Confirming only 1 Account');
+        List<Opportunity> oppListFromAcc = OpportunityService.getOpportunitiesByAccountId(oppsFromAcc[0].Id);
+        System.assertEquals(oppsFromAcc[0].Opportunities.size(), oppListFromAcc.size(), 'Confirming test and get count of accounts are equal');
```

Comment:
This message is misleading since you're actually checking the number of opportunities, not the number of accounts.  Update the message to match what you're actually looking for.